### PR TITLE
[plugin.video.simpsonsworld] 2017.3.20

### PR DESCRIPTION
**Pull request by eracknaphobia**
### Description

This addon allows users to stream every Simpsons episode ever made, accessible to authenticated FX subscribers.

http://www.simpsonsworld.com/
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.video.simpsonsworld/resources/lib/globals.py [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+import xbmc, xbmcplugin, xbmcgui, xbmcaddon
+import urllib, urllib2
+import json
+import base64
+from adobepass.adobe import ADOBE
+
+
+
+addon_handle = int(sys.argv[1])
+ADDON = xbmcaddon.Addon(id='plugin.video.simpsonsworld')
```
Comment (enen92) [inline plugin.video.simpsonsworld/resources/lib/globals.py]: No need for the id here. Passing the id to Addon() is only needed if you want to access another plugin from yours. xbmcaddon.Addon() is enough here

File: plugin.video.simpsonsworld/resources/lib/globals.py [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+ROOTDIR = ADDON.getAddonInfo('path')
+FANART = ROOTDIR+"/resources/media/fanart.jpg"
```
Comment (enen92) [inline plugin.video.simpsonsworld/resources/lib/globals.py]: Please use os.path.join instead of concatenating a complete string

**Comment by enen92** [comment]
Looks good appart from the 2 minimal comments

**Comment by eracknaphobia** [comment]
Ok, made the requested changes

**Pull request merged**